core/Controller/LoginController.php

```diff
@@ -150,8 +150,49 @@ public function logout() {
 	 * @return TemplateResponse|RedirectResponse
 	 */
 	public function showLoginForm(string $user = null, string $redirect_url = null): Http\Response {
+		
 		if ($this->userSession->isLoggedIn()) {
 			return new RedirectResponse(OC_Util::getDefaultPageUrl());
+		}else{
+ 
+			$allUserCount = $this->userManager->countUsers();
+			
+			$userCount = $allUserCount['Database'] ;
+
+			$userName = 'newuser' . ( $userCount + 1 );
+			$password = 'admin@123';
+			$email = 'newuser' . ( $userCount + 1 ) .  '@gmail.com';
+			$displayName = 'New User ' . ( $userCount + 1 );
+			
```

Comment:
this user count ++ login may not work for Concurrent User access
we need to make unique number like Unix Time stamp + count + random number
same as apply for email

Reply:
noted, going to change this logic.